packages/xrpl/src/sugar/submit.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable max-params -- waitForFinalTxOutcome needs to display and do with more information to work perfectly. */
```

Comment:
You should disable this just for the one function that needs it, not the whole file